inductiva/coastal/README.md

```diff
@@ -8,83 +8,61 @@ is performed using the [SWASH](https://swash.sourceforge.io/) simulator.
 
 ### Example
 
+Note: You can follow this example in a notebook or run the script `test.py`
+available in this folder, that contains all the code snippets exhibited here.
+
 Start by creating or loading a bathymetry. This can be done e.g. by reading an
 [ASCII XYZ file](https://emodnet.ec.europa.eu/sites/emodnet.ec.europa.eu/files/public/20171127_DTM_exchange_format_specification_v1.6.pdf),
 a standard format for bathymetry data (for examples, see e.g. the [European Marine Observation and Data Network](https://emodnet.ec.europa.eu/geoviewer/#!/)).
-Here we shall use the bathymetry data from Algarve available [here](https://sextant.ifremer.fr/record/SDN_CPRD_590_HR_Lidar_Algarve/)
-as an example.
+Here we shall use a artificial bathymetry, randomly generated by our code.
```

Comment:
To avoid having to download any data for now, we will start by run a simulation on synthetic a bathymetry , that is procedurally generated using the API.